app/views/timed_service_price_policies/_adjustment_row.html.haml

```diff
@@ -0,0 +1 @@
+= render "time_based_price_policies/adjustment_row", price_group: price_group, price_policy: price_policy, cancellation: false, minimum_cost: false
```

Comment:
If the default for `cancellation` and `minimum` cost are essentially `false`, do we need to include them here? I would probably accept the argument that this is more explicit.